src/list/FilterSidebar.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Label, SearchInput, CheckboxGroup, IndeterminateCheckbox } from '@sailthru/stui-components';
+import { fontSizeSmall, fontSizeMedium, fontSizeLarge, fontWeightBold, grayTint90 } from '@sailthru/design-tokens';
+
+const FilterContainer = styled.div`
+    margin-top: 1.875em;
+    padding: 1em;
+    height: 100vh;
+    background-color: ${grayTint90};
+    font-size: ${fontSizeMedium};
+`;
+
+const OptionGroupContainer = styled.div`
+    margin-top: 1em;
+    font-size: ${fontSizeMedium};
+`;
+
+export default function FilterSidebar({ onChange }) {
+    const [ searchValue, setSearchValue ] = useState('');
+    const [ selectedType, setSelectedType ] = useState([]);
+    const [ typesList, setTypesList ] = useState([]);
+
+    const types = ['Signup', 'Manage', 'Other'];
+
+    function onChangeType({newValue}) {
+        setSelectedType(newValue);
+        const typeList = [];
+        newValue.forEach(value => {
+            typeList.push(types[value]);
+        })
+        setTypesList(typeList);
+        onChange({types: typeList, name: searchValue});
```

Comment:
nit-pick: `typeList` vs `typesList` is hard to distinguish in the logic
```suggestion
        const typesFilter = [];
        newValue.forEach(value => {
            typesFilter.push(types[value]);
        })
        setTypesList(typesFilter);
        onChange({types: typesFilter, name: searchValue});
```